### ISSUE TITLE
CLIP - No progression in Training

### ISSUE THREAD

**Issue by TEnsorTHiru** [openai/CLIP#362]

**Comment by Crawler995** [comment]
hello, I am trying to compress the image encoder and fine-tune it to recover the accuracy. And I face a similar problem to you that the training loss gradually reduces to 4.13 (using ~200 iterations), after it, the loss no longer reduces. This is so strange.

**Comment by TEnsorTHiru** [comment]
> hello, I am trying to compress the image encoder and fine-tune it to recover the accuracy. And I face a similar problem to you that the training loss gradually reduces to 4.13 (using ~200 iterations), after it, the loss no longer reduces. This is so strange.

If you have solved it, please LMK. I still face this issue and IDK why?

**Comment by 502dxceit** [comment]
I can't find optim.zero_grad() after loss.backward(), have you cleaned the grad after backpropagation?

**Comment by tengshaofeng** [comment]
@Crawler995 @502dxceit @TEnsorTHiru @longouyang @welinder this is my training code, everything gose well. https://github.com/tengshaofeng/finetune-jina-clip-v2
